design-proposals/edge-infra-day2-improvements.md

```diff
@@ -0,0 +1,402 @@
+# Design Proposal: Edge Infrastructure Manager Day 2 Improvements
```

Comment:
Do we expect user to be able to create upgrades? How will we be able to verify upgrades that are not yet released ?

Reply:
What do you mean?
An advanced user can still create new OSProfiles, if they want to test new or custom EMT version.
Also, users can create OSUpdatePolicy to add new packages to Ubuntu/Mutable OSes.

Reply:
ok thanks for the clarification